scenario/state.py

```diff
@@ -767,18 +769,37 @@ def _to_ops(self) -> pebble.Notice:
 
 
 @dataclasses.dataclass(frozen=True)
-class _BoundNotice(_max_posargs(0)):
-    notice: Notice
-    container: "Container"
+class Check(_max_posargs(1)):
```

Comment:
Should this be named `CheckInfo` instead of `Check` (to correspond to `pebble.CheckInfo`)? We normally use the same class names as in Ops, right?

Reply:
Ah, yes, you're right. I originally had this as a much more minimal object but then later realised it basically had to be the same as `pebble.CheckInfo` since that gets returned from the `.info` property.